### PR TITLE
Add new recipe for gptools

### PR DESCRIPTION
**Pull request by smithsp**

### CODE REVIEW

File: recipes/gptools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    Full package documentation is located at http://gptools.readthedocs.org/
+
+    A printable PDF is available at https://media.readthedocs.org/pdf/gptools/latest/gptools.pdf
+
+    Releases are available in PyPI at https://pypi.python.org/pypi/gptools/
+
+    To install, simply execute::
+
+        pip install gptools
```
Comment (CJ-Wright) [inline recipes/gptools/meta.yaml]: You might consider lightly editing this description.

**Comment by smithsp** [comment]
@CJ-Wright Thanks for the feedback.

File: recipes/gptools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    - corner
+    - toolchain
+
+test:
+  imports:
+    - gptools
+
+about:
+  home: http://gptools.readthedocs.org/
+  license: GPL
```
Comment (synapticarbors) [inline recipes/gptools/meta.yaml]: `GPL-3.0`

File: recipes/gptools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    - setuptools
+    - cython
+    - numpy
+    - scipy
+    - matplotlib
+    - mpmath
+    - emcee
+    - corner
+    - toolchain
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
```
Comment (synapticarbors) [inline recipes/gptools/meta.yaml]: remove comments.

File: recipes/gptools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/gptools/meta.yaml]: I think only `python`, `setuptools`, `toolchain` and `numpy` are required to build the package. The rest look to be runtime requirements. 

Also see the following about building against numpy:
https://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy

File: recipes/gptools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    - scipy
+    - matplotlib
+    - mpmath
+    - emcee
+    - corner
+    - toolchain
+
+test:
+  imports:
+    - gptools
```
Comment (synapticarbors) [inline recipes/gptools/meta.yaml]: Also test importing `gptools.kernel`

**Comment by smithsp** [comment]
Thanks @synapticarbors

**Comment by pkgw** [comment]
The issue with the `corner` dep should now be resolved.

File: recipes/gptools/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  license_family: GPL
+  license_file: LICENSE
+  summary: 'Gaussian process regression with derivative constraints and predictions.'
+
+  description: |
+    Gaussian processes with arbitrary derivative constraints and predictions.
+
+    `gptools` is a Python package that provides a convenient, powerful, and extensible implementation of Gaussian process regression (GPR). Central to `gptool`'s implementation is support for derivatives and their variances. Furthermore, the implementation supports the incorporation of arbitrary linearly transformed quantities into the GP.
+
+    Developed and tested using Python 2.7 and scipy 0.14.0. May work with other versions, but it has not been tested under such configurations.
```
Comment (jakirkham) [inline recipes/gptools/meta.yaml]: nit: Might want to wrap these a bit more so it is easier to read on one page.

**Comment by smithsp** [comment]
@synapticarbors @CJ-Wright @jakirkham @pkgw 
Thanks for your help with this recipe.

**Pull request merged**